src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js

```diff
@@ -0,0 +1,130 @@
+                errorMessage: errorMessage,
+            };
+        }
+
+        validateEmail(event) {
+            const isEmpty =  !event.target.value.trim();
+            const isValid = global.eZ.errors.emailRegexp.test(event.target.value);
+            const isError = event.target.required && isEmpty || !isValid;
+            const fieldNode = event.target.closest(SELECTOR_FIELD_EMAIL);
+            let errorMessage;
+
+            if (isEmpty) {
+                errorMessage = global.eZ.errors.emptyField.replace('{fieldName}', fieldNode.querySelector(SELECTOR_FORM_LABEL).innerHTML);
+            } else if (!isValid) {
+                errorMessage = global.eZ.errors.invalidEmail;
+            }
+
+            return {
+                isError: isError,
+                errorMessage: errorMessage,
+            };
+        }
+
+        setIndex(parentNode, template) {return template.replace(/__index__/g, parentNode.querySelectorAll(SELECTOR_AUTHOR_ROW).length)}
+
+        updateDisabledState(parentNode) {
+            const isEnabled = parentNode.querySelectorAll(SELECTOR_AUTHOR_ROW).length > 1;
+
+            [...parentNode.querySelectorAll(SELECTOR_REMOVE_AUTHOR)].forEach(btn => {
+                if (isEnabled) {
+                    btn.removeAttribute('disabled');
+                } else {
+                    btn.setAttribute('disabled', true);
+                }
+            });
+        }
+
+        removeItem(event) {
+            const authorNode = event.target.closest(SELECTOR_FIELD);
+
+            event.target.closest(SELECTOR_AUTHOR_ROW).remove();
+
+            this.updateDisabledState(authorNode);
+        }
+
+        addItem(event) {
+            const authorNode = event.target.closest(SELECTOR_FIELD);
+            const template = authorNode.dataset.prototype;
+            const node = event.target.closest('.ez-field-edit-ui');
+
+            node.insertAdjacentHTML('beforeend', this.setIndex(authorNode, template));
+
+            this.reinit();
+            this.updateDisabledState(authorNode);
+        }
+
+        findStateNodes(fieldNode, input, selectors) {
+            const authorRow = input.closest(SELECTOR_AUTHOR_ROW);
+            const nodes = [
+                fieldNode,
+                authorRow
+            ];
+
+            return selectors.reduce((total, selector) => total.concat([...authorRow.querySelectorAll(selector)]), nodes);
+        }
+
+        findErrorNodes(fieldNode, input, selectors) {
+            const authorRow = input.closest(SELECTOR_AUTHOR_ROW);
+            const nodes = [
+                fieldNode,
+                authorRow
+            ];
+
+            return selectors.reduce((total, selector) => total.concat([...authorRow.querySelectorAll(selector)]), nodes);
+        }
+
+        findExistingErrorNodes(fieldNode, input, selectors) {
+            return selectors.reduce((total, selector) => total.concat([...input.closest(SELECTOR_AUTHOR_ROW).querySelectorAll(selector)]), []);
```

Comment:
I believe, you don't override this method. Just provide: `input.closest(SELECTOR_AUTHOR_ROW)` as your `fieldNode` variable :)
Compare with: https://github.com/ezsystems/ezplatform-admin-ui/pull/4/files#diff-0a93a220cf5748ddba0c72bcc10c42bdR20